ivy/exceptions.py

```diff
@@ -24,6 +24,28 @@ def __init__(self, message=""):
         super().__init__(message)
 
 
+class IvyIndex(IndexError, IvyBackendException):
```

Comment:
can we rename this to `IvyError`, then make this inheriting from `IndexError`, `ValueError` and `IvyException` (`IvyBackendException` is not needed), so that it's more unifying and extendable if there is similar case for another class